instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/assertions/RoadObject.kt

```diff
@@ -0,0 +1,5 @@
+package com.mapbox.navigation.instrumentation_tests.utils.assertions
+
+import com.mapbox.navigation.base.trip.model.roadobject.RoadObject
+
+fun RoadObject.compareIdWithIncidentId(incidentId: String): Boolean = id.contains(incidentId)
```

Comment:
NIT: compare implies it will return -1, 0 or 1. Maybe sth like `matchesIncidentId`?